script.js

```diff
@@ -9,3 +9,12 @@
 // Add a `facial-hair` property to `instructor`.
 
 // Log the value of the `facial-hair` property to the console.
+
+var colors = ["red","yellow","green"];
+console.log("green");
```

Comment:
not quite, how could you represent that by accessing the last item out of the array `colors`